edit-post/api/components/plugin-sidebar/sidebar-layout.js

```diff
@@ -0,0 +1,31 @@
+import './style.scss';
+
+/**
+ * WordPress dependencies
+ */
+import { IconButton } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const SidebarLayout = ( { onClose, title, children } ) => {
+	return (
+		<div
+			className="edit-post-sidebar edit-post-plugins-panel"
```

Comment:
https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#naming

Reply:
If we need to inherit styles from another component, we shouldn't simply assign the same class, we should create a separate component which is generalized to support both its current other usage, and the usage we're wanting here.